README.md

```diff
@@ -242,7 +240,7 @@ const ethMarketSnapshot = marketSnapshots.market[SupportedMarket.eth]
 const ethUserMarketSnapshot = marketSnapshots.user[SupportedMarket.eth]
 
 sdk.markets.write.modifyPosition({
-  marketAddress: ethMarketSnapshot.market,
+  marketAddress: ethMarketSnapshot.marketAddress,
```

Comment:
I wonder if we should change these writers and builders to take the `SupportedMarket` rather than the address, let's make a followup task to refactor that